index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main id="main"> 
+        <div id="img-div">
+          <figure>
+          <img id="image" src="https://i0.wp.com/www.themarginalian.org/wp-content/uploads/2014/01/alanwatts.jpg?w=680&ssl=1">
```

Comment:
I think it would be great if you can format your code so the lines aren't too long and difficult to read.